build/resolve_buck.py

```diff
@@ -72,6 +73,10 @@ class BuckInfo:
             "029b0bcc6f8e399185c1d0f574eba204934722b5",
         ],
     ),
+    ("linux", "aarch64"): BuckInfo(
+        archive_name="buck2-aarch64-unknown-linux-gnu.zst",
+        target_versions=["2024-05-15"],
```

Comment:
Thanks for taking a look at this. To confirm, is this the string returned by `buck2 --version`? I just want to confirm since the other binaries return a hash for the version.

Reply:
No, it's returning 
`buck2 49670bee56a7d8a7696409ca6fbf7551d2469787 <build-id>`
I couldn't find the hash on the buck2 github download page, so used the version string to download the version.
Will fix.

Reply:
Not sure how I missed the comments in the code how to add new versions. Fixed now.

Reply:
Thanks. Looks good to me.